lib/components/PoolUI.jsx

```diff
@@ -32,6 +33,26 @@ const renderErrorMessage = (address, type, message) => {
   poolToast.error(errorMsg)
 }
 
+// Jotai Atoms
+export const ethBalanceAtom = atom(ethers.utils.bigNumberify(0))
```

Comment:
What do you think of storing these `atom()` definitions in something like `lib/atoms/index`? I think it would be good practice to define something used by many different components in it's own place.

Reply:
Depends where we draw the line on "many different components". That index could get bloated if we do 2 or more components use the atom & keeping them inside the lowest level parent could be cleaner.

Reply:
I was suggesting something like `lib/atoms/index` but it could be `lib/atoms/poolAtoms.js`, etc.

If we only ever use simple atoms then it's fine as is. If we have any atoms that are more similar to redux, etc then we'll be blowing up those component files.

Here's some examples of more custom atoms:

```
const fetchUrlAtom = atom(
  async get => {
    const response = await fetch(get(urlAtom))
    return await response.json()
  }
)
```

```
const decrementCountAtom = atom(
  get => get(countAtom),
  (get, set, _arg) => set(countAtom, get(countAtom) - 1),
)
```

Either way we can always change it later!